in_toto/util.py

```diff
@@ -124,7 +124,7 @@ def prompt_import_rsa_key_from_file(filepath):
   password = None
   try:
     import_rsa_key_from_file(filepath)
-  except securesystemslib.exceptions.CryptoError, e:
+  except securesystemslib.exceptions.CryptoError:
```

Comment:
Correct. Loading an encrypted key without a password is just one scenario that triggers a `CryptoError`. When I wrote this I quickly wanted a user interface to load both encrypted and non-encrypted keys. 

We should probably re-factor the whole util module at some point and see if we can move functionality to  `securesystemslib.keys`. I suggest we do this in a separate PR though.